src/java/org/apache/cassandra/db/ClusteringBuilder.java

```diff
@@ -28,9 +28,9 @@
 /**
  * Allows to build ClusteringPrefixes, either Clustering or ClusteringBound.
  */
-public abstract class CBuilder
```

Comment:
Hi, I prefer to make these sorts of changes in a dedicated issue, so it looks like we can significantly reduce the number of changed files if we revert it, and keep only necessary changes here.

See the discussion below, we can ignore code formatting changes by listing the relevant commit hashes in a special file (this is not used by the project right now). So if we do this change in another issue and put that commit hash in a special file, it won't pollute the git blame in our IDEs.
https://github.com/orgs/community/discussions/5033
https://git-scm.com/docs/git-blame#Documentation/git-blame.txt-blameignoreRevsFile

This is my humble argument for such not making such a change here :-)